client/src/components/WorkflowInvocationState/Export/BioComputeObjectExportCard.vue

```diff
@@ -0,0 +1,174 @@
+                        submitting a BCO from Galaxy are available
+                        <a href="https://w3id.org/biocompute/tutorials/galaxy_quick_start/" target="_blank">here</a>.
+                    </p>
+                    <form @submit.prevent="submitForm">
+                        <div class="form-group">
+                            <label for="fetch">BCO DB URL (example: https://biocomputeobject.org)</label>
+                            <input
+                                id="fetch"
+                                v-model="form.fetch"
+                                type="text"
+                                class="form-control"
+                                placeholder="https://biocomputeobject.org"
+                                required />
+                        </div>
+                        <div class="form-group">
+                            <label for="authorization">User API Key</label>
+                            <input
+                                id="authorization"
+                                v-model="form.authorization"
+                                type="password"
+                                class="form-control"
+                                required />
+                        </div>
+                        <div class="form-group">
+                            <label for="table">Prefix</label>
+                            <input
+                                id="table"
+                                v-model="form.table"
+                                type="text"
+                                class="form-control"
+                                placeholder="BCO"
+                                required />
+                        </div>
+                        <div class="form-group">
+                            <label for="owner_group">User Name</label>
+                            <input
+                                id="owner_group"
+                                v-model="form.owner_group"
+                                type="text"
+                                class="form-control"
+                                required />
+                        </div>
+                        <div class="form-group">
+                            <button class="btn btn-primary">{{ "Submit" | localize }}</button>
+                        </div>
+                    </form>
+                </div>
+            </b-tab>
+        </b-tabs>
+    </div>
+</template>
+
+<script>
+import { getRootFromIndexLink } from "onload";
+import { getAppRoot } from "onload/loadConfig";
+import { mapGetters } from "vuex";
+import axios from "axios";
+
+const getUrl = (path) => getRootFromIndexLink() + path;
+
+export default {
+    props: {
+        invocationId: {
+            type: String,
+            required: true,
+        },
+    },
+    data() {
+        return {
+            bco: {
+                type: Object,
+                required: true,
+            },
```

Comment:
You can directly set this to null
```suggestion
            bco: null,
```